modules/file_upload/cms_file_upload.e

```diff
@@ -1,248 +0,0 @@
-note
-	description: "file_upload application root class"
-	date: "$Date$"
-	revision: "$Revision$"
-
-class
-	CMS_FILE_UPLOAD
-
-inherit
-	CMS_MODULE
-		redefine
-			install,
-			initialize,
-			setup_hooks,
-			permissions
-		end
-
-	CMS_HOOK_BLOCK
-
-	CMS_HOOK_MENU_SYSTEM_ALTER
-
-	SHARED_EXECUTION_ENVIRONMENT
-
-create
-	make
-
-feature {NONE} -- Initialization
-
-	make
-		do
-			name := "file_uploader"
-			version := "1.0"
-			description := "Service to upload some files"
-			package := "file upload"
-		end
-
-feature -- Access
-
-	name: STRING
-
-	permissions: LIST [READABLE_STRING_8]
-			-- List of permission ids, used by this module, and declared.
-		do
-			Result := Precursor
-			Result.force ("manage admin")
-			Result.force ("admin users")
-			Result.force ("admin roles")
-			Result.force ("admin modules")
-			Result.force ("install modules")
-			Result.force ("admin core caches")
-			Result.force ("clear blocks cache")
-			Result.force ("admin export")
-			Result.force ("export core")
-
-			Result.force ("upload files")
-		end
-
-feature {CMS_API} -- Module Initialization
-
-	initialize (api: CMS_API)
-			-- <Precursor>
-		do
-			Precursor (api)
-		end
-
-feature {CMS_API}-- Module management
-
-	install (api: CMS_API)
-			-- install the module
-		local
-			sql: STRING
-		do
-			-- create a database table
-			if attached {CMS_STORAGE_SQL_I} api.storage as l_sql_storage then
-				if not l_sql_storage.sql_table_exists ("file_upload_table") then
-					sql := "[
-CREATE TABLE file_upload_table(
-  `id` INTEGER PRIMARY KEY AUTO_INCREMENT NOT NULL CHECK("id">=0),
-  `name` VARCHAR(100) NOT NULL,
-  `uploaded_date` DATE,
-  `size` INTEGER
-);
-					]"
-					l_sql_storage.sql_execute_script (sql, Void)
-					if l_sql_storage.has_error then
-						api.logger.put_error ("Could not initialize database for file uploader module", generating_type)
-					end
-				end
-				Precursor {CMS_MODULE}(api)
-			end
-		end
-
-feature -- Access: router
-
-
-	setup_router (a_router: WSF_ROUTER; a_api: CMS_API)
-			-- <Precursor>
-		local
-			www: WSF_FILE_SYSTEM_HANDLER
-		do
-
-			map_uri_template_agent (a_router, "/upload/", agent execute_upload, void)
-
-			create www.make_with_path (document_root)
```

Comment:
not needed, such handler is already registered for any /files/*  requests.
which includes  /files/uploaded_files/*